includes/Traits/VendorAuthorizable.php

```
@@ -13,4 +16,83 @@ trait VendorAuthorizable {
+     * - A valid vendor user, or
+     * - A vendor staff member with a valid associated vendor.
+     *
+     * Used for REST API validation callbacks.
+     *
+     * @param mixed          $value   The value to validate.
+     * @param \WP_REST_Request $request The REST API request object.
+     * @param string         $key     The parameter key.
+     *
+     * @return bool|\WP_Error True if valid, WP_Error if invalid.
+     */
+    public function validate_store_id( $value, $request, $key ) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$request'. (undefined)

(UnusedFormalParameter)